docs/docs/installation_guide/installation/pre-installation/conf-settings.md

```diff
@@ -61,6 +59,14 @@ The names of many deployed resources include a namespace string such as `mystage
 + Region short name (for example: `va` for US-East-1, or for US-West-2, defined in `.defaults.yml`)
 + Solution name
 
+### Other Beijing/Ningxia configurations
```

Comment:
Other configurations for Beijing/Ningxia regions

Reply:
Refactord